### PR TITLE
Cybits Chain to Blockcore-Hub

### PR DESCRIPTION
**Pull request by psavva**
This PR add the Cybits chain to the blockcore-hub

### CODE REVIEW

**Comment by psavva** [comment]
There is also an improvement to the background alignment - centre when changing screen sizes.

File: src/app/services/chain.service.ts [hunk 1]
```diff
@@ -63,7 +63,9 @@ export class ChainService {
...
-                { opreturndata: 80, pos: true, rootFolderName: 'rsc', test: false, name: 'Royal Sports City', unit: 'RSC', coin: 'RSC', chain: 'RSC', private: 122, public: 60, identity: 'RoyalSportsCity', tooltip: 'Royal Sports City', port: 14001, rpcPort: 14002, apiPort: 14003, wsPort: 4337, network: 'RoyalSportsCity', genesisDate: new Date(2022, 0, 5) }
+                { opreturndata: 80, pos: true, rootFolderName: 'rsc', test: false, name: 'Royal Sports City', unit: 'RSC', coin: 'RSC', chain: 'RSC', private: 122, public: 60, identity: 'RoyalSportsCity', tooltip: 'Royal Sports City', port: 14001, rpcPort: 14002, apiPort: 14003, wsPort: 4337, network: 'RoyalSportsCity', genesisDate: new Date(2022, 0, 5) },
+                { opreturndata: 80, pos: true, rootFolderName: 'Cybits', test: false, name: 'Cybits Main', chain: 'Cybits', private: 160, public: 28, identity: 'Cybits', tooltip: 'Cybits', port: 17771, rpcPort: 17772, apiPort: 17773, network: 'CybitsMain', genesisDate: new Date(Date.UTC(2021, 10, 1)) },
```
Comment (sondreb) [inline src/app/services/chain.service.ts]: Are you sure the root folder name is with capital C? It must be correct casing or fails on Linux.

Comment (sondreb) [reply src/app/services/chain.service.ts]: Don't call the mainnet for "Main" this is displayed to users.

**Review by sondreb — CHANGES_REQUESTED** [review]
Thanks for the PR, please look at the capitalization of root folder name.

**Comment by psavva** [comment]
Good Morning @sondreb 
Changes have been incorporated.

File: src/app/services/chain.service.ts (hunk 1 above)
Comment (psavva) [reply src/app/services/chain.service.ts]: Done.

**Review by sondreb — APPROVED** [review]
Thanks for the PR!

**Pull request merged**